### PR TITLE
Added class rooms, hover animations, some styling work

### LINKED ISSUE #2: Create E,F,G,H classrooms pages
Requirements:
- [x] a template for showing classrooms of a specific letter
- [x] should be responsive, mobile & desktop
- [x] letter of classrooms should be in url, e.g. `tc.gdscyu.com/classrooms/E`

### PR DESCRIPTION
**Pull request by ghost**
Closes #2

### CODE REVIEW

File: src/components/ClassroomButton.astro [hunk 1]
```diff
@@ -13,24 +13,34 @@ interface Props {
...
 const { text, color, href } = Astro.props;
-
+let colorHover: string;
 let colorClass: string;
+/* hover colors to be changed*/
 switch (color) {
   case ClassroomButtonColor.paleYellow:
-    colorClass = "bg-pale-yellow";
+    colorClass = "bg-pale-yellow ";
+    colorHover = "hover:text-orange ";
```
Comment (YazeedAlKhalaf) [inline src/components/ClassroomButton.astro:22]: it is all the same, why not make it a const above or just put it directly in the html code?

File: src/components/ClassroomButton.astro [hunk 2]
```diff
@@ -13,24 +13,34 @@ interface Props {
 }
 
 const { text, color, href } = Astro.props;
-
+let colorHover: string;
 let colorClass: string;
+/* hover colors to be changed*/
```
Comment (YazeedAlKhalaf) [inline src/components/ClassroomButton.astro]: it is better if you choose them now so we do not forget. ask the designers for them,

**Review by YazeedAlKhalaf — CHANGES_REQUESTED** [review]
great but can you finalize the colors?

**Comment by ghost** [comment]
closes #10

**Review by YazeedAlKhalaf — APPROVED** [review]
LGTM

**Pull request merged**